### ISSUE TITLE
thumbs-up or down - related to the content or to the existence of the content?

### ISSUE THREAD

**Issue by Mannshoch** [friendica/friendica#12723]
How should I use Like / Dislike?

Is that meant about the content itself or the importance of the content?
I have here content that tells about a bad case. But the Author which post it do tell his opinion about the content, so I do not know If he agrees with it or not. I could only assume it because I know the Account. 

In this current case, should I Like it and say, the content is important. Or should I Dislike it and value the case described?

**Comment by annando** [comment]
Let's imagine that someone writes that their cat just died. Would you want to like or dislike that post?

**Comment by Mannshoch** [comment]
I would dislike it because it's not tactful, to like the death of something.

**Comment by Mannshoch** [comment]
It's my first real use of social media. I avoided Facebook and other until now. I'm not sure what's common.

**Comment by MrPetovan** [comment]
I use Likes as read receipts when I don't have anything meaningful to answer. Since I don't like the Dislike feature, I would leave a Like on the pet funeral post.

**Comment by MrPetovan** [comment]
Please note that the Dislike feature is only possibly shown to other Friendica users, as it is a Friendica-specific optional feature.

**Comment by AlfredSK** [comment]
Can't say anything about dislike - I have disabled it. The meaning of a 'dislike' is too ambiguous for me.
I like posts/comments when I want to express that I like the content. When a 'like' is not enough to express myself I prefer to write comment.

**Comment by Mannshoch** [comment]
In my opinion I would prefer an "Important content" or "not important content" button.
The use of "Like" for "I have read it" is also a common use at my work. But the wording "like" feels so awful wrong. May that's because I'm not a native English speaker.

In context to Mastodon. I hear sometime "boosting content" I supposed that a like or dislike will do that. Am I wrong?

**Comment by MrPetovan** [comment]
Boosting means resharing. I believe Likes are called "Favorites" on Mastodon.

**Comment by annando** [comment]
@MrPetovan dislikes are also used on Lemmy as "downvotes".

**Comment by foss-** [comment]
The question asked is a very valid one. I took a long journey from thinking dislikes are really important because if I can *like* something I should also be able to *dislike*.

Having used Friendica and other social media for many years I have arrived at the conclusion that dislikes add nothing of value to a conversation.

Then again I do enjoy the fact I can pick any emoji to respond e.g. to a Signal message. But being limited to a thumbs down, I think considering removing that feature is probably worth having a discussion about.

**Comment by MrPetovan** [comment]
Emoji reactions would be valuable but we aren't there yet, especially for picking one on the web frontend. 😅

**Comment by annando** [comment]
In fact we are already processing the emoji reactions from Pleroma. We only need to display them.

**Comment by SpcCw** [comment]
From the very beginning I treated likes as "thank you for sharing this" and reshares as "hey everyone, look what I found". This is how I use and interpret them and neither mean I like or support the actual content. This is what comments are for (although emoji reactions are pretty neat too).

I also don't use dislikes because I don't think they add anything useful. Basically it is like saying "why are you bringing this up here, shut up" and this is not polite. Votes and downvotes can be useful on forums there they indicate relevance or success rate of an answer but I am not sure they are really healthy on social networks. If I dislike something I scroll past.

**Comment by annando** [comment]
@SpcCw since we federate with Lemmy, it is good that we can like/dislike (up-/downvote)

**Comment by MrPetovan** [comment]
It's only marginally good, since we don't end up having the sorting on up/downvotes in Friendica. What's left is a very optional feature on Friendica that makes everybody uncomfortable and that mostly doesn't show (disabled on the most popular nodes...).

I'm still advocating for its complete removal.

**Comment by annando** [comment]
Since we already can disable it and there is some use for it, I strongly advocate against it. Also it is part of the AP protocol.

**Comment by Quix0r** [comment]
I use the dislike against trolls and others who want to annoy me, also I may use it when I disagree but don't want to comment on it. Strong vote in favor of this feature, please don't kill existing features only because of it may make people feel uncomfortable.

**Comment by MrPetovan** [comment]
You don't understand, it doesn't make the author of the posts you add a dislike to uncomfortable, it makes the Friendica users themselves uncomfortable when presented with the feature, which is pretty bad. That's two people in two days who have been asking the same question about the use of Dislikes.

If not for making our own users uncomfortable (even though it's a good reason on its own), we should remove this feature at least to respect the time of the people who have to field these questions.

**Comment by Quix0r** [comment]
No, keep the feature in. Softies shouldn't dictate us what features we should have. And only their emotions are hurt (by a dislike, seriously ...) then they can still deactivate it. #SafeTheDislike ?

**Comment by SpcCw** [comment]
I also don't think it should be removed. It is just a part of toolset and it can have its uses right now as well is in the future. E.g. someone might want to run a group where likes and dislikes are treated as voting.

I am not sure what the defaults are but if it is on maybe it should be turned off by default because it is extra feature and sometimes causes confusion and adds to the learning curve. But please don't remove it completely :)

**Comment by foss-** [comment]
I'd rather see proper polls supported than thumbs up / down used for voting, but that's a different discussion.

**Comment by SpcCw** [comment]
@foss- in my humble opinion polls and likes/dislikes are different tools meant to use in different scenarios and that's why many systems (not necessarily Fedi) provide both.

Possible cases: the content is generated by some RSS or some other automation and local members want to vote "yes/no", a thread with the top post describing a problem/issue and comments are suggestions which are voted up/down based on their success rate, etc...

Anyway it is not a Friendica-specific feature, it is part of AP and other platforms use it with possibility of new ones emerging. Removing or restricting it will _reduce compatibility_ and it is not desirable in my opinion.

**Comment by MrPetovan** [comment]
Likes/Dislikes should be rendered obsolete by emoji reactions, not polls. Even if we end up mapping 👍 to "Like" and 👎 to "Dislike" internally for protocol communication.

**Comment by Mannshoch** [comment]
I like the idea of up vote and down vote. But It has to be clear in what context it is meant.  Using "Like" as kind of "have read it and I wish you know it", create a kind of distance into it and the idea of a like as a short comment what I think shrink it's value.

**Comment by MrPetovan** [comment]
Upvotes and Downvotes have specific meaning in forum-like software where they’re use to improve the relevance of displayed content. For example on StackOverflow it helps displaying the most popular answer first, regardless of when it was submitted.

There’s no such sorting in Friendica so the meaning is lost.

**Comment by SpcCw** [comment]
> There’s no such sorting in Friendica so the meaning is lost.

There are visible counters which also can be useful in this context, at least for the author.

Emoji reactions are basically extension of likes/dislikes and they can also be used for same purposes and rise same questions. E.g. someone posts about a political event and gets angry face emoji reaction. Was it their reaction to the event? Or was someone displeased with the post about it?

In my opinion all such reactions (and even short comments) can come with some uncertainty but it is expected and not a big deal. I remember that when we had no likes (in blogs like LiveJournal) some people posted dots in comments. Just a single "." - which essentially had meaning of the modern like - "saw this post, can't/don't want say much but thanks for sharing". And some people hated this too :) Such is life.

**Comment by MrPetovan** [comment]
I didn't say it couldn't be useful, just that the meaning imparted to dislikes/downvotes by an explicit sorting mechanism is lost in Friendica and makes the feature ambiguous which prompted a few questions about it.

**Comment by loma-one** [comment]
I don't understand this discussion at this point. If the not liking should be removed, then consequently the liking must also be deleted. Both functions are subject to an interpretation that is made by the users and cannot be decided by developers. There is no 1 or 0

Liking something is also interpreted by many users as "I have taken positive note of it, but will not post a comment". Just as not liking something indicates a rejection of a post, as well as expressing a negative, positive attitude.

Do I like a contribution when, for example, a report from Ukraine is distributed, in which special circumstances are reported? Is at the point a not like more appropriate, because I want to express my discomfort, but do not want to leave a comment. I definitely don't want to like it, so I use a don't like.

**Comment by MrPetovan** [comment]
Again, likes and dislikes may sound like the same feature on paper but they aren’t equally ambiguous. Otherwise we would be getting a similar amount of support requests for both features, which isn’t the case.

**Comment by informapirata** [comment]
In commercial social networks such as facebook and youtube, the dislike can create many problems for the users who suffer it: in fact, dislikes can induce insecurity and self-esteem crises in the victim and lead to a decrease in the visibility of the post and of the profile that published it. Also they can be used to bully the user.
In a social place where narcissism is an important component, this can cause serious damage.
In the fediverse, on the other hand, it is very difficult for these cases to occur. Moreover:
- friendica (and lemmy) communities are tiny. They are even tiny compared to the mastodon community, the largest in the fediverse, yet so residual in absolute terms!
- in the fediverse there are almost never narcissistic intentions and the subjects who use it have always acquired a good solidity of character, thanks to a greater awareness of digital social relationships
- the user can also understand thanks to the dislikes if the instance in which he is located is actually the most suitable one.

The only problem with dislikes is the polarizing effect that can occur during long discussions of an oppositional nature, with different fan groups "voting" for the user they support...
But it would be foolish to think of totally eliminating the conflict from the fediverse: indeed it is important that the conflict is not sterilized a priori, but that the possibility of exercising it at low intensity continues to exist, to ensure that we do not stop producing antibodies that will be useful in case of hostile and toxic conflicts.
